src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2731,8 +2731,10 @@ because xbuild doesn't support framework reference assemblies.
     PlatformToolsVersion="$(AndroidSdkPlatformToolsVersion)"
     NdkVersion="$(AndroidNdkVersion)"
     NdkRequired="$(_NdkRequired)"
+    JdkVersion="$(JavaSdkVersion)"
   >
     <Output TaskParameter="Dependencies" ItemName="AndroidDependency" />
+    <Output TaskParameter="JavaDependencies" ItemName="JavaDependency" />
```

Comment:
Looping in @tondat to help review with respect to the IDE and SDK Manager. Using a new `@(JavaDependency)` output item should hopefully reduce the chance of potential breakage in other areas that depend on `GetAndroidDependencies`

Reply:
We found [a couple of places](https://github.com/xamarin/XamarinVS/blob/313d332b37180718cbf77864d2afbc38b8c027bc/src/Core/VisualStudio.Android/Services/AndroidToolsBuildService.cs#L87) where the return value seems to be used without name / itemspec filtering, so I am going to try to remove `@(JavaDependency)` from the target return value for now.

Reply:
@dellis1972 @jonathanpeppers given the potential breakage with IDE/SDK Manager, I am leaning towards adding a separate `GetJavaDependencies` target that [`<InstallAndroidDependencies/>`](https://github.com/xamarin/android-sdk-installer/blob/68939f222b0e794424123ce6cb46bf0ba7ef73a4/Xamarin.Installer.Build.Tasks/Xamarin.Installer.Common.targets#L11) can rely on, does that approach seem reasonable?

Reply:
To keep the changes smaller, you could also do a new MSBuild property? `-p:AndroidReturnJavaDependencies=true`

The existing `InstallAndroidDependencies` target would set this, but the existing IDEs wouldn't:

```xml
<Target Name="_SetAndroidReturnJavaDependencies">
  <PropertyGroup>
    <AndroidReturnJavaDependencies Condition=" '$(AndroidReturnJavaDependencies)' == '' ">true</AndroidReturnJavaDependencies>
  </PropertyGroup>
</Target>

<Target Name="InstallAndroidDependencies" DependsOnTargets="_SetAndroidReturnJavaDependencies;GetAndroidDependencies">
<!-- ... -->
```
Then your `<CalculateProjectDependencies/>` task would only return Java stuff if `$(AndroidReturnJavaDependencies)` was true.

I think either option would work, it's either invent a new property or new target?

Reply:
Added a property to opt out of this new item by default which should hopefully address the concerns of external dependencies.